src/model/entities/rundown.ts

```diff
@@ -543,6 +546,159 @@ export class Rundown extends BasicRundown {
     this.markNextPart()
   }
 
+  public setNextFromDirection(direction: SetNextDirection, owner?: Owner): void {
+    this.assertActive(this.setNextFromDirection.name)
+    const nextCursor: RundownCursor = this.findNextCursorFromDirection(direction)
+    this.setNextFromIds(nextCursor.segment.id, nextCursor.part.id, owner)
+  }
+
+  private findNextCursorFromDirection(direction: SetNextDirection): RundownCursor {
+    switch (direction) {
+      case SetNextDirection.PART_AFTER_NEXT_PART: {
+        return this.findCursorWithPartAfterCurrentNextPart()
+      }
+      case SetNextDirection.PART_BEFORE_NEXT_PART: {
+        return this.findCursorWithPartBeforeCurrentNextPart()
+      }
+      case SetNextDirection.SEGMENT_BEFORE_NEXT_SEGMENT: {
+        return this.findCursorWithFirstPartInSegmentBeforeNextSegment()
+      }
+      case SetNextDirection.SEGMENT_AFTER_NEXT_SEGMENT: {
+        return this.findCursorWithFirstPartInSegmentAfterNextSegment()
+      }
+    }
+  }
+
+  private findCursorWithPartAfterCurrentNextPart(): RundownCursor {
+    this.assertNotUndefined(this.nextCursor, 'next Cursor')
+    const currentNextSegment: Segment = this.nextCursor.segment
+    const currentNextPart: Part = this.nextCursor.part
+    try {
+      const nextPart: Part = currentNextSegment.findNextPartNotOnAir(currentNextPart)
+      return this.createCursorFromSegmentAndPart(currentNextSegment, nextPart)
+    }  catch (exception) {
```

Comment:
```suggestion
    } catch (exception) {
```

There's something with the formatting not having been run on this file :D It's not a problem, just wanted to say that there are formatter things that could be applied :)

Reply:
I can confidently say that formatting has been run. I run `yarn test` and `yarn lint-fix` before every commit ;) 
As to why the formatting didn't catch this, I don't know...
(Just did a `yarn lint-fix` again on this branch to very it didn't catch it)

Reply:
Nevertheless, the space is now gone :)